management/server/account.go

```diff
@@ -471,6 +475,15 @@ func (am *DefaultAccountManager) handleGroupsPropagationSettings(ctx context.Con
 	}
 }
 
+func (am *DefaultAccountManager) handleAutoUpdateVersionSettings(ctx context.Context, oldSettings, newSettings *types.Settings, userID, accountID string) {
+	if oldSettings.AutoUpdateVersion != newSettings.AutoUpdateVersion {
+		am.StoreEvent(ctx, userID, accountID, accountID, activity.AccountAutoUpdateVersionUpdated, map[string]any{
+			"old_value": oldSettings.AutoUpdateVersion,
+			"new_value": newSettings.AutoUpdateVersion,
+		})
+	}
+}
```

Comment:
I think we  could simplifying the meta to just store current version instead of both `old_value` and `new_value`. The event sequence already captures the change history, so we don't need the redundancy.
 
 ```suggestion
func (am *DefaultAccountManager) handleAutoUpdateVersionSettings(ctx context.Context, oldSettings, newSettings *types.Settings, userID, accountID string) {
	if oldSettings.AutoUpdateVersion != newSettings.AutoUpdateVersion {
		am.StoreEvent(ctx, userID, accountID, accountID, activity.AccountAutoUpdateVersionUpdated, map[string]any{
			"version": newSettings.AutoUpdateVersion,
		})
	}
}
```